torchrl/envs/common.py

```diff
@@ -624,6 +676,7 @@ def __getattr__(self, attr: str) -> Any:
         elif "_env" in self.__dir__():
             env = self.__getattribute__("_env")
             return getattr(env, attr)
+        super().__getattr__(attr)
```

Comment:
there is not return here
I think you should move that to ParallelEnv.__getattribute__

Reply:
But the error occurs in TransformedEnv and it only inherit from _EnvClass. Added the return i still getting the same error ( Also added it to ParrallelEnv)